sdk/src/main/java/com/meniga/sdk/MenigaSettings.java

```diff
@@ -51,6 +51,7 @@
 
 	private MenigaSettings(Builder builder) {
 		endpoint = builder.endpoint;
+		timeout = builder.timeout <= 0 ? 60 : builder.timeout;
```

Comment:
Sure thing.

Reply:
Actually this is needed because special endpoint definitions that do not specify a different timeout need to have the general timeout from the settings, not default 60s. That can only be achieved by either having timeout nullable (requiring a check here anyway) or zero/negative. Negative timeouts are illegal anyway and should not be allowed.